[2022-07-19 11:27:11,632] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:26:00+00:00 [queued]>
[2022-07-19 11:27:11,643] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:26:00+00:00 [queued]>
[2022-07-19 11:27:11,643] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 11:27:11,644] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 11:27:11,645] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 11:27:11,665] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 11:26:00+00:00
[2022-07-19 11:27:11,671] {standard_task_runner.py:52} INFO - Started process 11193 to run task
[2022-07-19 11:27:11,676] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T11:26:00+00:00', '--job-id', '580', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpn3a_s2h2', '--error-file', '/tmp/tmpkxbboib0']
[2022-07-19 11:27:11,677] {standard_task_runner.py:80} INFO - Job 580: Subtask respond
[2022-07-19 11:27:11,770] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T11:26:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 11:27:11,889] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T11:26:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T11:26:00+00:00
[2022-07-19 11:27:21,058] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 11:27:21,072] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 11:27:21,219] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T112600, start_date=20220719T112711, end_date=20220719T112721
[2022-07-19 11:27:21,299] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 11:27:21,485] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
